### ISSUE TITLE
Scarlet Monastery Cathedral Corrupted Ashbringer Event

### ISSUE THREAD

**Issue by Ludevic** [dalaranwow/dalaran-wow#5255]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The event while wearing the Corrupted Ashbringer inside SM Cath does not work properly.
**Expected behaviour**: Tell us what should happen instead
While you walk inside an event is supposed to start as shown here http://wowwiki.wikia.com/wiki/Corrupted_Ashbringer/Scarlet_Monastery_Event
**Steps to reproduce the problem**:

1.  Equip Corrupted Ashbringer
2.  Enter Sm Cath
3.  

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Your issue posted here is a duplicate of issue: XXXXXhttps://github.com/dalaranwow/dalaran-wow/issues/2517 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
